pageserver/src/tenant/timeline.rs

```diff
@@ -4768,6 +4789,10 @@ impl<'a> TimelineWriter<'a> {
     pub(crate) fn update_current_logical_size(&self, delta: i64) {
         self.tl.update_current_logical_size(delta)
     }
+
+    pub(crate) fn update_directory_entries_count(&self, kind: DirectoryKind, count: u64) {
+        self.tl.update_directory_entries_count(kind, count);
+    }
```

Comment:
Note the writer derefs as Timeline.

Reply:
IDK there is a similar function `update_current_logical_size` right above. But I've removed it, shrug.

Reply:
Yeah this is... what IT is.